### PR TITLE
Merging template updates 2.7.1

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting please

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting

File: nextflow.config [hunk 1]
```diff
@@ -276,12 +282,12 @@ dag {
...
     name            = 'nf-core/sarek'
-    author          = 'Maxime Garcia, Szilveszter Juhos, Friederike Hanssen'
+    author          = """Maxime Garcia, Szilveszter Juhos, Friederike Hanssen"""
     homePage        = 'https://github.com/nf-core/sarek'
-    description     = 'An open-source analysis pipeline to detect germline or somatic variants from whole genome or targeted sequencing'
+    description     = """An open-source analysis pipeline to detect germline or somatic variants from whole genome or targeted sequencing"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=21.10.3'
-    version = '3.2dev'
+    nextflowVersion = '!>=22.10.1'
```
Comment (FriederikeHanssen) [inline nextflow.config:289]:
```suggestion
    nextflowVersion = '!>=21.10.3'
```

File: docs/usage.md [hunk 2]
```diff
@@ -402,8 +401,14 @@ Work dir:
 Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`
 ```
 
+#### For beginners
+
+A first step to bypass this error, you could try to increase the amount of CPUs, memory, and time for the whole pipeline. Therefor you can try to increase the resource for the parameters `--max_cpus`, `--max_memory`, and `--max_time`. Based on the error above, you have to increase the amount of memory. Therefore you can go to the [parameter documentation of rnaseq](https://nf-co.re/rnaseq/3.9/parameters) and scroll down to the `show hidden parameter` button to get the default value for `--max_memory`. In this case 128GB, you than can try to run your pipeline again with `--max_memory 200GB -resume` to skip all process, that were already calculated. If you can not increase the resource of the complete pipeline, you can try to adapt the resource for a single process as mentioned below.
```
Comment (maxulysse) [inline docs/usage.md:406]:
```suggestion
A first step to bypass this error, you could try to increase the amount of CPUs, memory, and time for the whole pipeline. Therefor you can try to increase the resource for the parameters `--max_cpus`, `--max_memory`, and `--max_time`. Based on the error above, you have to increase the amount of memory.
Therefore you can go to the [parameter documentation of sarek](https://nf-co.re/sarek/3.1.1/parameters) and scroll down to the `show hidden parameter` button to get the default value for `--max_memory`.
In this case 128GB, you than can try to run your pipeline again with `--max_memory 200GB -resume` to skip all process, that were already calculated.
If you can not increase the resource of the complete pipeline, you can try to adapt the resource for a single process as mentioned below.
```

**Review by maxulysse — CHANGES_REQUESTED** [review]
rnaseq -> sarek

**Review by maxulysse — APPROVED** [review]

**Pull request merged**